test-cases/longevity/longevity-large-partition-2days.yaml

```diff
@@ -0,0 +1,41 @@
+test_duration: 3000
+
+bench_run: true
+
+prepare_write_cmd:  ["scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=8 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=16 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=8 -clustering-row-count=50000 -partition-offset=24 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=32 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=39 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=46 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=7 -clustering-row-count=50000 -partition-offset=53 -clustering-row-size=uniform:100..5120 -concurrency=250 -rows-per-request=2000 -timeout=180s -connection-count 150"
+]
+stress_cmd: ["scylla-bench -workload=uniform -mode=read -select-order-by=asc -replication-factor=3 -partition-count=60 -clustering-row-count=50000 -clustering-row-size=uniform:100..5120 -rows-per-request=2000 -timeout=240s -retry-number=10 -concurrency=400 -duration=44h -connection-count 400"
+]
+
+n_db_nodes: 4
+n_loaders: 2
+n_monitor_nodes: 1
+
+instance_type_db: 'i4i.4xlarge'
+
+nemesis_class_name: 'SisyphusMonkey'
+nemesis_seed: '001'
+instance_type_loader: 'm5n.2xlarge'
```

Comment:
looks like it it didn't help to overcome the issue in the loader, so change it back to c5n.

Reply:
fixed